apps/docs/docs/template/api/virtual-scrolling.md

```diff
@@ -0,0 +1,862 @@
+| `item$`       | `Observable<T>`                                                | the same value as $implicit, but as `Observable`                                                                                                                                                                  |
+| `index$`      | `Observable<number>`                                           | index as `Observable`                                                                                                                                                                                             |
+| `count$`      | `Observable<number>`                                           | count as `Observable`                                                                                                                                                                                             |
+| `first$`      | `Observable<boolean>`                                          | first as `Observable`                                                                                                                                                                                             |
+| `last$`       | `Observable<boolean>`                                          | last as `Observable`                                                                                                                                                                                              |
+| `even$`       | `Observable<boolean>`                                          | even as `Observable`                                                                                                                                                                                              |
+| `odd$`        | `Observable<boolean>`                                          | odd as `Observable`                                                                                                                                                                                               |
+| `select`      | `(keys: (keyof T)[], distinctByMap) => Observable<Partial<T>>` | returns a selection function which accepts an array of properties to pluck out of every list item. The function returns the selected properties of the current list item as distinct `Observable` key-value-pair. |
+
+**Usage**
+
+```html
+<rx-virtual-scroll-viewport [itemSize]="50">
+  <div
+    *rxVirtualFor="
+      let item of observableItems$; trackBy: 'id';
+      let count$ = count$;
+      let index$ = index$;
+      let first$ = first$;
+      let last$ = last$;
+      let even$ = even$;
+      let odd$ = odd$;
+    "
+  >
+    <div *rxLet="count$; let c">{{ c }}</div>
+    <!---->
+  </div>
+</rx-virtual-scroll-viewport>
+```
+
+## Advanced Usage
+
+### Use render strategies (`strategy`)
```

Comment:
Something exactly like this section we need for Fixed size virtual-scroll using `*rxVirtualFor` with X

Reply:
I don't understand?